.github/workflows/changelog-entry-check.yml

```diff
@@ -1,29 +1,29 @@
 name: Changelog entry check
 
 on:
-    pull_request:
-        types:
-        -   opened
-        -   reopened
-        -   labeled
-        -   unlabeled
-        -   synchronize
+  pull_request:
+    types:
```

Comment:
Do you know why this list gets indented whereas lists in other files, e.g. `pre-commit-config.yaml` align the bullets with the parent node?